lib/stdlib/doc/src/binary.xml

```diff
@@ -256,6 +256,25 @@
       </desc>
     </func>
 
+    <func>
+      <name name="encode_hex" arity="2" since="OTP 26.0"/>
```

Comment:
I have updated this line to  `<name name="encode_hex" arity="2" since=""/>`. I don't know how to give a placeholder here. I don't find anywhere mention of the `placeholder` in [CONTRIBUTING.md](https://github.com/erlang/otp/blob/master/CONTRIBUTING.md)

Reply:
You can't put it there at this point because you don't know yet. It is a ticket number that is internal to the OTP team, AFAIK, and will eventually be given to you by the person a PR is assigned to.